src/app/layout.tsx

```diff
@@ -1,8 +1,8 @@
 import { CssBaseline } from '@mui/material'
 import type { Metadata } from 'next'
 import { Roboto } from 'next/font/google'
-import Header from '../components/Header/Header'
-import { Footer } from '../components'
+import { Footer, Header} from '../components'
+import './global.css'
 
 //TODO for create fonts configuration variable.
 const roboto = Roboto({
```

Comment:
i guess it can remove this Roboto from line 8, right?

Reply:
that why I need to discuss with you, I do not know who did this way, but we to talk about global css.